N°4765 - Update brazilian translations

eduardomozart opened the pull request:
Removed the ``~~`` flag from translated strings.

eduardomozart committed:
Updated PT-BR Translation

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -33,15 +33,15 @@
 	'Core:AttributePhoneNumber+' => '',
 
-	'Core:AttributeObsolescenceDate' => 'Obsolescência data',
+	'Core:AttributeObsolescenceDate' => 'Data de obsolescência ',
```
```suggestion
	'Core:AttributeObsolescenceDate' => 'Data de obsolescência',
```

Molkobain commented on dictionaries/pt_br.dictionary.itop.ui.php:
```diff
@@ -368,30 +368,30 @@
 	'UI:Button:Search' => ' Pesquisar ',
-	'UI:Button:Clear' => ' Clear ~~',
-	'UI:Button:SearchInHierarchy' => 'Search in hierarchy~~',
+	'UI:Button:Clear' => ' Limpar ',
```
```suggestion
	'UI:Button:Clear' => ' Limpar',
```

Molkobain reviewed (changes requested):
Thanks for the PR! Can you just accept the 2 changes I suggested? (you can click accept directly on the webpage, no need to do another PR)

Molkobain committed:
Fix typo

Molkobain committed:
Fix typo

Molkobain commented:
Nevermind, I was able to apply them myself.
I'll discuss this PR internally and get back to you soon.

piRGoif commented:
Hello,
Thanks for your contribution !

I see multiple "+" keys containing same values as their counterparts... For example Class:ActionEmail/Attribute:importance/Value:low and Class:ActionEmail/Attribute:importance/Value:low+.
This is the same in the EN files :/ @Molkobain what do you think ? Shouldn't we fix that ?

Hipska commented:
> I see multiple "+" keys containing same values as their counterparts... For example Class:ActionEmail/Attribute:importance/Value:low and Class:ActionEmail/Attribute:importance/Value:low+. This is the same in the EN files :/ @Molkobain what do you think ? Shouldn't we fix that ?

I would say, yes!

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -254,10 +254,10 @@
+	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objkey' => 'ID do objeto',
+	'Class:CMDBChangeOp/Attribute:objkey+' => 'ID do objeto',
```
```suggestion
	'Class:CMDBChangeOp/Attribute:objkey+' => '',
```

Molkobain committed:
Remove unnecessary attribute+ entry

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -254,10 +254,10 @@
-	'Class:CMDBChangeOp/Attribute:objkey' => 'ID objeto',
-	'Class:CMDBChangeOp/Attribute:objkey+' => 'ID objeto',
+	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe do objeto',
```
```suggestion
	'Class:CMDBChangeOp/Attribute:objclass+' => '',
```

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -470,10 +470,10 @@
-	'Class:EventLoginUsage/Attribute:contact_email' => 'Email usuário',
-	'Class:EventLoginUsage/Attribute:contact_email+' => 'Endereço email deste usuário',
+	'Class:EventLoginUsage/Attribute:contact_name' => 'Nome de usuário',
+	'Class:EventLoginUsage/Attribute:contact_name+' => 'Nome de usuário',
```
```suggestion
	'Class:EventLoginUsage/Attribute:contact_name+' => '',
```

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -516,40 +516,40 @@
+	'Class:ActionEmail/Attribute:importance' => 'Prioridade',
+	'Class:ActionEmail/Attribute:importance+' => 'Flag de prioridade',
+	'Class:ActionEmail/Attribute:importance/Value:low' => 'baixa',
+	'Class:ActionEmail/Attribute:importance/Value:low+' => 'baixa',
```
```suggestion
	'Class:ActionEmail/Attribute:importance/Value:low+' => '',
```

Molkobain commented on dictionaries/pt_br.dictionary.itop.core.php:
```diff
@@ -516,40 +516,40 @@
-	'Class:ActionEmail/Attribute:importance/Value:high' => 'alto',
-	'Class:ActionEmail/Attribute:importance/Value:high+' => 'alto',
+	'Class:ActionEmail/Attribute:importance/Value:high' => 'alta',
+	'Class:ActionEmail/Attribute:importance/Value:high+' => 'alta',
```
```suggestion
	'Class:ActionEmail/Attribute:importance/Value:high+' => '',
```

Molkobain committed:
Remove unnecessary attribute+ entry

Molkobain committed:
Remove unnecessary attribute+ entry

Molkobain committed:
Remove unnecessary attribute+ entry

Molkobain committed:
Remove unnecessary attribute+ entry

Molkobain commented:
@eduardomozart I took the liberty to remove entries for attribute descriptions that were identical to the attribute names. It's a best pratice now but they were indeed already present in the files you translated. :)

eduardomozart committed:
Update brazilian translations

eduardomozart committed:
Merge branch 'develop' of https://github.com/eduardomozart/iTop into develop

eduardomozart committed:
Fix missing space

eduardomozart committed:
Update DB fetch term

eduardomozart commented:
Hello guys,
I updated some strings from the iTop translation to Portuguese-Brazil during this weekend. Apparently most of the strings were translated with Google Translate at the time, since most strings lacked congruence, especially those that make up the table headers (which justifies this commit having so many translated strings updated). I also adjusted some terms to the Brazilian terms used in the study material and ITIL 4 exam in Brazil.
I intend to continue updating the translation as I use iTop and identify inconsistencies in the translation, as well as the release of new versions of iTop, becoming part of the iTop translation group. I intend to translate modules and extensions as I implement them in my environment.
Thank you for developing iTop, I hope to contribute with bug reports and new features to iTop.

eduardomozart committed:
Update pt_br.dictionary.itop.ui.php

Dashboard panel translation update

Hipska commented:
That's great to hear! Currently our (Super-Visions, partner) extensions to iTop are completely translated to Portuguese, but if you would like to sign up, you will be informed if new strings need to be translated: https://www.transifex.com/super-visions/public/

eduardomozart committed:
Update translation

eduardomozart committed:
Update brazilian translation

eduardomozart committed:
Fix notifications dropdown menu translation text

The translated strings for the "Status" menu isn't displayed correctly into the Notifications because it's string isn't recognised.

eduardomozart commented:
The Pull Request "[Fix notifications dropdown menu translation text](https://github.com/Combodo/iTop/pull/268/commits/c4aaabaeea8c9286cb92ec82366c32943e8c3835)" ended up being sent in this same Pull Request, which was not my intention. During the translation work, I found that the text in the "Status" drop-down menu of iTop Notifications was not displayed correctly, instead the translated but generic "Status" of other objects was displayed. By removing the "+" character from the notification parameters, the translated terms started to be displayed correctly. I made the change in the other language files.

eduardomozart committed:
Update brazilian translation

eduardomozart committed:
Updated bulk delete term

Hipska commented:
Please mark this PR as draft asl long as you are still working on it.

piRGoif commented:
Hello,

> Please mark this PR as draft asl long as you are still working on it.

Yes please :)
And when you are ready for us to review your pr, add a comment stating so.

Thanks !

eduardomozart committed:
Updated Brazilian translation

Molkobain commented on datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php:
```diff
@@ -33,7 +33,7 @@
 	'Attachment:Max_Mo' => '(Tamanho máximo de arquivo: %1$s Mb)',
 	'Attachment:Max_Ko' => '(Tamanho máximo de arquivo: %1$s Kb)',
-	'Attachments:NoAttachment' => 'Nenhum anexo. ',
+	'Attachments:NoAttachment' => 'Nenhum anexo ',
```
```suggestion
	'Attachments:NoAttachment' => 'Nenhum anexo',
```

Molkobain replied:
@eduardomozart please accept this change so we can merge your PR.
You can also check the "allow maintainers to edit PR" on the PR so we can apply changes directly :)

eduardomozart replied:
@Molkobain this merge was already applied to the PR and "Allow edits by maintainers" was already enable for this PR. Please let me know if anymore changes are needed.

Molkobain replied:
Indeed it was fixed in the meantime, but was not reflected on my GitHub GUI. Thanks!

piRGoif commented:
> The Pull Request "[Fix notifications dropdown menu translation text](https://github.com/Combodo/iTop/pull/268/commits/c4aaabaeea8c9286cb92ec82366c32943e8c3835)" ended up being sent in this same Pull Request, which was not my intention. During the translation work, I found that the text in the "Status" drop-down menu of iTop Notifications was not displayed correctly, instead the translated but generic "Status" of other objects was displayed. By removing the "+" character from the notification parameters, the translated terms started to be displayed correctly. I made the change in the other language files.

By "itop notification" do you mean the bell icon on the bottom left of the UI ? 
Can you share some screenshots of the wrong behavior ?

eduardomozart committed:
Updated brazilian translation

eduardomozart committed:
Translate Notifications GUI

eduardomozart commented:
Hello @piRGoif 
The requested screenshots are in attachment:

Before the fix:

![before-fix](https://user-images.githubusercontent.com/2974895/154123256-ba131abe-f3fe-475c-9d98-32ae38af4064.png)

After the fix:

![after-fix](https://user-images.githubusercontent.com/2974895/154123250-afe3e865-4059-471e-93ac-4fbeadf77a74.png)
.

eduardomozart committed:
Small translation change

Molkobain commented:
Hello Eduardo,

Product team is ok to include the translations in the future 3.0.2 of iTop. Do you think you will have finished what you want to translate by then? (After summer)

eduardomozart committed:
Update translation

eduardomozart commented:
Hello,
Sorry for the late reply. Yes, the language file is near from complete and can be merged. Please close this pull request so I can pull the repo again and update newer strings added between 3.0.1/3.0.2 release.

eduardomozart commented:
Hello.

Why this translation was considered invalid? There was something missing?

piRGoif commented:
Hello,
You closed this PR on may 9 ? You said :

> Please close this pull request so I can pull the repo again and update newer strings added between 3.0.1/3.0.2 release.

The "invalid" label for us means precisely that : the PR cannot be merged and will be replaced by something else.

Did you create a new PR ? Do you wish to restart work on this one ?

eduardomozart committed:
Merge branch 'develop' into develop

eduardomozart commented:
Sorry, I closed this to say it was finished, not to end it as it is. I fixed the inconsistences and now it can be safely merged. Tested this translation at our production environment and it seems to be working just fine. Hope to see this translation into next iTop major version, as it will greatly improve PT-BR experience for Portuguese users. We can continue using this PR. Thank you!

piRGoif commented:
Oh ok ! Close is a final state, opposed to merge (a close pr is unmerged)
Thanks for updating your fork branch !
We will review your modifications !

Molkobain commented:
Accepted in functional review, will be part of the upcoming 3.1.0

Molkobain commented:
Hello @eduardomozart , translations will part of the upcoming 3.1.0; thanks again for the contribution! 🙌
Could you send me your fullname and postal address so we can send you contributor stickers?

Cheers,
Guillaume

eduardomozart commented:
Hello @Molkobain, thank you! Of course I can. Here's my address information (but there's any problem if you can't send it to my country):

Full Name: Eduardo Mozart de Oliveira
Address: ******
City: *******
State: ***
Zip Code: ******
Country: ********

I pretend to keep contributing with iTop with translations and backport some GLPI plugins (like ["accounts"](https://github.com/InfotelGLPI/accounts) plugin) as an iTop extension and offer an iTop course at Udemy at the near future.

_Edited by Molkobain to obfuscate personal address_

Molkobain commented:
Thanks! I'll send them this week, keep up the good work! 🤩🤩🤩
